### PR TITLE
docs(math/base/ops): improve `math/base/ops` README.md examples

### LINKED ISSUE #1569: [RFC]: improve README examples of `math/base/ops` namespace
### Description

This RFC proposes to improve the [README examples](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/ops#examples) of the `math/base/ops` namespace package. Currently, namespace package READMEs only include minimal examples which are not particularly informative and do not showcase namespace functionality.

Historically, namespace examples were not given much effort due to the rapidly changing nature of the project; however, now that development and organization has settled, it would be good to revisit these packages and add proper examples to better communicate to users how to use namespace contents.

For those wanting to contribute to this issue, you can use the following steps:

- Browse through the [contents](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/ops) of the `math/base/ops` namespace and identify interesting functions and packages to showcase.
- Update the example code block in the "Examples" section of the README for the `math/base/ops` namespace and open a PR.
- When updating the README, examples should follow existing project conventions (e.g., ES5).
- Examples may `require` functionality from other packages to help make examples more interesting.

### Related Issues

None.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by EuniceSim142**
Resolves #1569 .

## Description

> What is the purpose of this pull request?

This pull request:

-   Adds examples for methods in `math/base/ops` namespace
-   Updates ReadMe.md with the same methods.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #1569
-   fixes #1569

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 1]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
 <!-- eslint no-undef: "error" -->
 
 ```javascript
+var Complex128 = require( '@stdlib/complex/float64' );
 var objectKeys = require( '@stdlib/utils/keys' );
 var ns = require( '@stdlib/math/base/ops' );
 
 console.log( objectKeys( ns ) );
+
+// Operations for floating point numbers (double-precision)
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
// Operations for double-precision floating point numbers:
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 2]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
...
+console.log( ns.add(1.25, 0.45) ); // => 1.7
+console.log( ns.sub(1.25, 0.45) ); // => 0.8
+
+// (single-precision)
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
// Operations for single-precision floating point numbers:
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 3]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
 <!-- eslint no-undef: "error" -->
 
 ```javascript
+var Complex128 = require( '@stdlib/complex/float64' );
 var objectKeys = require( '@stdlib/utils/keys' );
 var ns = require( '@stdlib/math/base/ops' );
 
 console.log( objectKeys( ns ) );
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]: Let's not print out the namespace contents anymore; that was mostly a placeholder:

```suggestion
var Complex128 = require( '@stdlib/complex/float64' );
var ns = require( '@stdlib/math/base/ops' );
```

Users can always browse the docs to see what packages are part of the namespace.

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 4]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
...
+
+// Operations for floating point numbers (double-precision)
+console.log( ns.add(1.25, 0.45) ); // => 1.7
+console.log( ns.sub(1.25, 0.45) ); // => 0.8
+
+// (single-precision)
+console.log( ns.mulf(1.3, 1.2) ); // => ~ 1.56
+console.log( ns.divf(1.2, 0.4) ); // => 3
+
+// Operations for complex numbers
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
// Operations for complex numbers:
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 5]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
...
 ```javascript
+var Complex128 = require( '@stdlib/complex/float64' );
 var objectKeys = require( '@stdlib/utils/keys' );
 var ns = require( '@stdlib/math/base/ops' );
 
 console.log( objectKeys( ns ) );
+
+// Operations for floating point numbers (double-precision)
+console.log( ns.add(1.25, 0.45) ); // => 1.7
+console.log( ns.sub(1.25, 0.45) ); // => 0.8
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]: Console print annotations of the form `// => ...` should always be placed on the following line:

```suggestion
console.log( ns.add( 1.25, 0.45 ) ); 
// => 1.7

console.log( ns.sub( 1.25, 0.45 ) ); 
// => 0.8
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 6]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
...
+
+// (single-precision)
+console.log( ns.mulf(1.3, 1.2) ); // => ~ 1.56
+console.log( ns.divf(1.2, 0.4) ); // => 3
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
console.log( ns.mulf( 1.3, 1.2 ) ); 
// => ~1.56

console.log( ns.divf( 1.2, 0.4 ) ); 
// => 3.0
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 7]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
...
+
+// Operations for complex numbers
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+
+console.log( ns.cmul( z1, z2 ) ); // => Complex128 { re: -13.0, im: -1.0 }
+
+// Operations for Signed 32-bit integers
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
// Operations for signed 32-bit integers:
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 8]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
...
+// 2^30 * -5 = -5368709120 => 32-bit integer overflow
+console.log( ns.imul( 1073741824|0, -5|0 ) ); // => -1073741824
+
+// Operations for Unsigned 32-bit integers
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
// Operations for unsigned 32-bit integers:
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 9]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
...
+// 2^31 * 5 = 10737418240 => 32-bit integer overflow
+console.log( ns.umul( 2147483648>>>0, 5>>>0 ) ); // => 2147483648
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
console.log( ns.umul( 2147483648>>>0, 5>>>0 ) );
// => 2147483648
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 10]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
...
+
+// Operations for Double word product:
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
// Operations for double word product:
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 11]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
...
+// -(2^31) * 2^30 = -2305843009213694000 => 32-bit integer overflow
+console.log( ns.imuldw( 0x80000000|0, 0x40000000|0 ) ); // => returns [ -536870912, 0 ]
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
console.log( ns.imuldw( 0x80000000|0, 0x40000000|0 ) );
// => [ -536870912, 0 ]
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 12]
```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
...
+
+// (single-precision)
+console.log( ns.mulf(1.3, 1.2) ); // => ~ 1.56
+console.log( ns.divf(1.2, 0.4) ); // => 3
+
+// Operations for complex numbers
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+
+console.log( ns.cmul( z1, z2 ) ); // => Complex128 { re: -13.0, im: -1.0 }
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
console.log( ns.cmul( z1, z2 ) ); 
// => Complex128 { re: -13.0, im: -1.0 }
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 13]
```diff
@@ -18,7 +18,34 @@
...
 var objectKeys = require( '@stdlib/utils/keys' );
 var ns = require( './../lib' );
 
 console.log( objectKeys( ns ) );
+
+// Operations for floating point numbers (double-precision)
+console.log( ns.add(1.25, 0.45) ); // => 1.7
+console.log( ns.sub(1.25, 0.45) ); // => 0.8
+
+// (single-precision)
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
// Operations for single-precision floating point numbers:
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 14]
```diff
@@ -18,7 +18,34 @@
 
 'use strict';
 
+var Complex128 = require( '@stdlib/complex/float64' );
 var objectKeys = require( '@stdlib/utils/keys' );
 var ns = require( './../lib' );
 
 console.log( objectKeys( ns ) );
+
+// Operations for floating point numbers (double-precision)
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
// Operations for double-precision floating point numbers:
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 15]
```diff
@@ -18,7 +18,34 @@
 
 'use strict';
 
+var Complex128 = require( '@stdlib/complex/float64' );
 var objectKeys = require( '@stdlib/utils/keys' );
 var ns = require( './../lib' );
 
 console.log( objectKeys( ns ) );
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
var Complex128 = require( '@stdlib/complex/float64' );
var ns = require( './../lib' );
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 16]
```diff
@@ -18,7 +18,34 @@
...
+
+// Operations for floating point numbers (double-precision)
+console.log( ns.add(1.25, 0.45) ); // => 1.7
+console.log( ns.sub(1.25, 0.45) ); // => 0.8
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
console.log( ns.add( 1.25, 0.45 ) );
// => 1.7

console.log( ns.sub( 1.25, 0.45 ) );
// => 0.8
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 17]
```diff
@@ -18,7 +18,34 @@
...
+console.log( ns.mulf(1.3, 1.2) ); // => ~ 1.56
+console.log( ns.divf(1.2, 0.4) ); // => 3
+
+// Operations for complex numbers
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+
+console.log( ns.cmul( z1, z2 ) ); // => Complex128 { re: -13.0, im: -1.0 }
+
+// Operations for Signed 32-bit integers
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
// Operations for signed 32-bit integers:
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 18]
```diff
@@ -18,7 +18,34 @@
...
+// 2^30 * -5 = -5368709120 => 32-bit integer overflow
+console.log( ns.imul( 1073741824|0, -5|0 ) ); // => -1073741824
+
+// Operations for Unsigned 32-bit integers
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
// Operations for unsigned 32-bit integers:
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 19]
```diff
@@ -18,7 +18,34 @@
...
+
+// Operations for complex numbers
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+
+console.log( ns.cmul( z1, z2 ) ); // => Complex128 { re: -13.0, im: -1.0 }
+
+// Operations for Signed 32-bit integers
+// 2^30 * -5 = -5368709120 => 32-bit integer overflow
+console.log( ns.imul( 1073741824|0, -5|0 ) ); // => -1073741824
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
console.log( ns.imul( 1073741824|0, -5|0 ) );
// => -1073741824
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 20]
```diff
@@ -18,7 +18,34 @@
...
+
+// Operations for Unsigned 32-bit integers
+// 2^31 * 5 = 10737418240 => 32-bit integer overflow
+console.log( ns.umul( 2147483648>>>0, 5>>>0 ) ); // => 2147483648
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
console.log( ns.umul( 2147483648>>>0, 5>>>0 ) );
// => 2147483648
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 21]
```diff
@@ -18,7 +18,34 @@
...
+
+// Operations for Double word product:
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
// Operations for couble word product:
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 22]
```diff
@@ -18,7 +18,34 @@
...
+// -(2^31) * 2^30 = -2305843009213694000 => 32-bit integer overflow
+console.log( ns.imuldw( 0x80000000|0, 0x40000000|0 ) ); // => returns [ -536870912, 0 ]
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
console.log( ns.imuldw( 0x80000000|0, 0x40000000|0 ) );
// => [ -536870912, 0 ]
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 23]
```diff
@@ -87,10 +87,43 @@ The namespace contains the following functions:
...
+// Operations for complex numbers:
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+
+console.log( ns.cmul( z1, z2 ) ); 
+// => Complex128 { re: -13.0, im: -1.0 }
+
+// Operations for signed 32-bit integers:
+// 2^30 * -5 = -5368709120 => 32-bit integer overflow
+console.log( ns.imul( 1073741824|0, -5|0 ) ); // => -1073741824
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
console.log( ns.imul( 1073741824|0, -5|0 ) );
// => -1073741824
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 24]
```diff
@@ -18,7 +18,37 @@
...
+// Operations for double-precision floating point numbers:
+console.log( ns.add( 1.25, 0.45 ) );
+// => 1.7
+
+console.log( ns.sub( 1.25, 0.45 ) );
+// => 0.8
+
+// Operations for single-precision floating point numbers:
+console.log( ns.mulf(1.3, 1.2) ); // => ~ 1.56
+console.log( ns.divf(1.2, 0.4) ); // => 3
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
console.log( ns.mulf( 1.3, 1.2 ) ); 
// => ~1.56

console.log( ns.divf( 1.2, 0.4 ) ); 
// => 3.0
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 25]
```diff
@@ -18,7 +18,37 @@
...
+
+// Operations for complex numbers
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
// Operations for complex numbers:
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 26]
```diff
@@ -18,7 +18,37 @@
...
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+
+console.log( ns.cmul( z1, z2 ) ); // => Complex128 { re: -13.0, im: -1.0 }
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
console.log( ns.cmul( z1, z2 ) );
// => Complex128 { re: -13.0, im: -1.0 }
```

**Review by Planeshifter — APPROVED** [review]
Thanks for your contribution! Will merge once CI has passed.

Made a few edits. Please be aware that comment annotations of the form `// =>` (for `console.log` printouts) or `// returns` (for return values) should always be included in the next line. This follows project conventions and also ensures that our linting tooling can verify that the annotations are correct.

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 27]
```diff
@@ -87,10 +87,44 @@ The namespace contains the following functions:
...
+console.log( ns.mulf( 1.3, 1.2 ) );
+// => ~1.56
+
+console.log( ns.divf( 1.2, 0.4 ) );
+// => 3.0
+
+// Operations for complex numbers:
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
```

**Comment by EuniceSim142** [comment]
Thank you for taking time to review! Will take note of these details in future 😄

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 28]
```diff
@@ -87,10 +87,43 @@ The namespace contains the following functions:
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var objectKeys = require( '@stdlib/utils/keys' );
+var Complex128 = require( '@stdlib/complex/float64' );
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md:90]:
```suggestion
var Complex128 = require( '@stdlib/complex/float64' );
var real = require( '@stdlib/complex/real' );
var imag = require( '@stdlib/complex/imag' );
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 29]
```diff
@@ -87,10 +87,43 @@ The namespace contains the following functions:
...
+// => ~1.56
+
+console.log( ns.divf( 1.2, 0.4 ) );
+// => 3.0
+
+// Operations for complex numbers:
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+console.log( ns.cmul( z1, z2 ) );
+// => Complex128 { re: -13.0, im: -1.0 }
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
// => <Complex128> { re: -13.0, im: -1.0 }
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 30]
```diff
@@ -18,7 +18,40 @@
...
+// => ~1.56
+
+console.log( ns.divf( 1.2, 0.4 ) );
+// => 3.0
+
+// Operations for complex numbers:
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+console.log( ns.cmul( z1, z2 ) );
+// => Complex128 { re: -13.0, im: -1.0 }
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
// => <Complex128> { re: -13.0, im: -1.0 }
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 31]
```diff
@@ -87,10 +87,43 @@ The namespace contains the following functions:
...
+// => ~1.56
+
+console.log( ns.divf( 1.2, 0.4 ) );
+// => 3.0
+
+// Operations for complex numbers:
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+console.log( ns.cmul( z1, z2 ) );
+// => <Complex128> { re: -13.0, im: -1.0 }
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
// => <Complex128>
// { 're': -13.0, 'im': -1.0 }
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 32]
```diff
@@ -18,7 +18,40 @@
...
+// => ~1.56
+
+console.log( ns.divf( 1.2, 0.4 ) );
+// => 3.0
+
+// Operations for complex numbers:
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+console.log( ns.cmul( z1, z2 ) );
+// => <Complex128> { re: -13.0, im: -1.0 }
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
console.log( ns.cmul( z1, z2 ) );
// => <Complex128> 
// {'re': -13.0, 'im': -1.0 }
```

File: lib/node_modules/@stdlib/math/base/ops/README.md [hunk 33]
```diff
@@ -87,10 +87,44 @@ The namespace contains the following functions:
...
+
+console.log( ns.divf( 1.2, 0.4 ) );
+// => 3.0
+
+// Operations for complex numbers:
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+console.log( ns.cmul( z1, z2 ) );
+// => <Complex128>
+// { 're': -13.0, 'im': -1.0 }
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/README.md]:
```suggestion
console.log( ns.cmul( z1, z2 ) ); // { 're': -13.0, 'im': -1.0 }
// => <Complex128>
```

File: lib/node_modules/@stdlib/math/base/ops/examples/index.js [hunk 34]
```diff
@@ -18,7 +18,41 @@
...
+
+console.log( ns.divf( 1.2, 0.4 ) );
+// => 3.0
+
+// Operations for complex numbers:
+var z1 = new Complex128( 5.0, 3.0 );
+var z2 = new Complex128( -2.0, 1.0 );
+console.log( ns.cmul( z1, z2 ) );
+// => <Complex128> 
+// {'re': -13.0, 'im': -1.0 }
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/ops/examples/index.js]:
```suggestion
console.log( ns.cmul( z1, z2 ) ); // {'re': -13.0, 'im': -1.0 }
// => <Complex128>
```

**Pull request merged**